orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -350,6 +529,7 @@ func buildBaseOrchestrator(network *configurator.Network, graph *configurator.En
 		ret["td-agent-bit"] = configs["td-agent-bit"]
 		ret["eventd"] = configs["eventd"]
 	}
+	ret["state"] = configs["state"]
```

Comment:
Move this into the initial map defn -- line 522+. No need to have it separate down here